pyresample/test/test_gradient.py

```diff
@@ -261,6 +263,313 @@ def test_resample_swath_to_area_3d(self):
             assert np.allclose(arr[np.isfinite(arr)], float(i + 1))
 
 
+class TestRBGradientSearchResamplerArea2Area:
+    """Test RBGradientSearchResampler for the Area to Area case."""
+
+    def setup(self):
+        """Set up the test case."""
+        from pyresample.gradient import ResampleBlocksGradientSearchResampler
+        self.src_area = AreaDefinition('src', 'src area', None,
+                                       {'ellps': 'WGS84', 'h': '35785831', 'proj': 'geos'},
+                                       100, 100,
+                                       (5550000.0, 5550000.0, -5550000.0, -5550000.0))
+
+        self.dst_area = AreaDefinition('euro40', 'euro40', None,
+                                       {'proj': 'stere', 'lon_0': 14.0,
+                                        'lat_0': 90.0, 'lat_ts': 60.0,
+                                        'ellps': 'bessel'},
+                                       102, 102,
+                                       (-2717181.7304994687, -5571048.14031214,
+                                        1378818.2695005313, -1475048.1403121399))
+
+        self.resampler = ResampleBlocksGradientSearchResampler(self.src_area, self.dst_area)
```

Comment:
Is this used? Pytest doesn't use a method called `setup` does it? And I don't see this being called.

https://docs.pytest.org/en/6.2.x/xunit_setup.html

Reply:
`setup` should be called (it works in other classes), but yeah, this class isn't used at the moment (no tests for this)

Reply:
removed

Reply:
Ok, thanks. As discussed on slack, `setup` is pytest's way of keeping things `nose` compatible, but it is is not a first-class feature (documented) of pytest. You should use `setup_method` in the future.